api/backend/ecs/load_balancer_manager.go

```diff
@@ -304,17 +303,23 @@ func (e *ECSLoadBalancerManager) createLoadBalancer(
 		return err
 	}
 
-	securityGroup, err := e.upsertSecurityGroup(ecsLoadBalancerID, ports)
-	if err != nil {
-		return err
+	// only public load balancers get an additional security group
```

Comment:
are there any side effects here for private load balancers when envs are linked?